APMrover2/sailboat.cpp

```diff
@@ -86,6 +86,24 @@ const AP_Param::GroupInfo Sailboat::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("WNDSPD_MIN", 7, Sailboat, sail_windspeed_min, 0),
 
+    // @Param: SAIL_MAX_X_TRACK
```

Comment:
sorry to be annoying but could we rename this to SAIL_XTRACK_MAX or something similar.  Normally we put "_MAX" at the end of the parameter name.  Also I think we can drop the "SAIL_" part from the parameter description.

Reply:
fixed